toggles/webapp/toggles.ts

```diff
@@ -54,11 +40,5 @@ export default {
       description:
         'Displays body copy in Helvetica regular (where it is currently Helvetica light)',
     },
```

Comment:
@gestchild do we still need `openWithAdvisoryPrototype`? Is it there as a protection, or was it there for us to test?

Reply:
it was left there as a protection, but probably safe to remove now

Reply:
Is it reliant on another service? And if so, do we handle failures gracefully?

Reply:
Maybe we can chat about this in another PR